index.ipynb

```diff
@@ -55,6 +55,17 @@
     "  * Exercise 2: Making a wave! **[5min]**\n",
     "\n",
     "* Session 2: [Seismic Imaging](seismic_imaging.ipynb)\n",
+    "    * Add receivers to record observations and build a complete forward operator **[5min]**\n",
+    "    * Exercise 3: Gradient operator **[5min]**\n",
+    "    * Intro to optimisation **[5min]**\n",
+    "        * Compare with machine learning, weather forecasting etc. minimise difference between observed and simulated.\n",
+    "        * Gradient descent. In a PDE constrained optimisation problem, the gradient is obtained by the adjoint-state method.\n",
```

Comment:
This is not correct, adjoint-state is not a PDE constrained optimization problem.

Reply:
"We will solve a PDE-constrained optimisation problem with a gradient-descent method, obtaining the gradient using the adjoint-state method."
Does the above rephrasing make you happier @mloubout ?

Reply:
Well no, that's the thing, in time-domain, we only solve the  "reduced problem" which is an unconstrained least-square problem (or other depending on the way you measure the misfit). The PDE constraint is eliminated due to computationally unsolvable to have the reduced unconstrained adjoint-state problem.

Reply:
So i would simply say
" We will solve an adjoint-state optimization problem with gradient descent for a simple seismic inversion problem"